google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITDirectPathFallback.java

```diff
@@ -110,7 +110,7 @@ public ITDirectPathFallback() {
   }
 
   @Before
-  public void setup() throws IOException, Throwable {
+  public void setup() throws Throwable {
```

Comment:
nit: Does this really need to be `throws Throwable`? Or could it throw something that is a little more specific?

Reply:
We did not need any `throws` actually, thanks for spotting this, fixed now.